python/cudf/cudf/core/column/column.py

```diff
@@ -701,11 +703,26 @@ def to_pandas(
             raise ValueError(
                 f"{arrow_type=} and {nullable=} cannot both be set."
             )
-        elif nullable:
-            raise NotImplementedError(f"{nullable=} is not implemented.")
         pa_array = self.to_arrow()
-        if arrow_type:
+
+        # Check if dtype is an ArrowDtype or pd.ExtensionDtype subclass
+        if arrow_type or isinstance(self.dtype, pd.ArrowDtype):
             return pd.Index(pd.arrays.ArrowExtensionArray(pa_array))
+        elif (
+            (
+                nullable
+                or isinstance(self.dtype, pd.core.dtypes.dtypes.ExtensionDtype)
```

Comment:
Could you always access `ExtensionDtype` from `pd.api.extensions`?

Reply:
Fixed it.